### ISSUE TITLE
Support labels in ModeSwitch

### ISSUE THREAD

**Issue by cassidyjames** [elementary/granite#332]
Instead of icons, we should be able to provide labels to a ModeSwitch to style it like this:

![Anamorph screenshot](https://raw.githubusercontent.com/cassidyjames/anamorph/master/data/screenshot.png)

Perhaps we could have it automatically fallback to the tooltip text if there's no icon provided. If that's the case we should probably rename the properties from assuming `icon` and `tooltip` so it's more clear/accurate.

**Comment by cassidyjames** [comment]
The smallest change for names would probably be:

* `primary_icon_tooltip_text` → `primary_text`
* `secondary_icon_tooltip_text` → `secondary_text`

…and then some internal variable names that assume it's an icon. Here's the code from Anamorph if it's useful at all:

https://github.com/cassidyjames/anamorph/blob/d8ffe51148d22be24de5a5daf7a234485444355a/src/Views/SetupView.vala#L51-L92

We'd likely want to [`ascii_up`](https://valadoc.org/glib-2.0/string.ascii_up.html) the strings automatically if we want to enforce that stylistically.

**Comment by fitojb** [comment]
> We'd likely want to `ascii_up` the strings automatically if we want to enforce that stylistically.

Please don’t do this; just provide the translatable strings in all caps and let localizers replicate it manually. The reason is that not languages like Irish may have other capitalization rules. See https://bugzilla.mozilla.org/show_bug.cgi?id=854756 for more information.

**Comment by cassidyjames** [comment]
@fitojb yeah, that's been brought up on Slack as well. The problem is we'd like to enforce it stylistically somehow, but it's difficult to do that. We can instead provide guidance in the HIG but it's better to enforce it somehow if possible.
